pkg/utils/cluster_info.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	clusterIDConfMap = "cluster-id"
+	// Todo: Is there a place where i can get this?
+	configMapNamespace = "liqo"
+)
+
+func GetClusterID(c client.Client) (string, error) {
+
+	configMaps := &corev1.ConfigMapList{}
+	// possibility to specify "Liqo" namespace if it is possible to get it somewhere
+	if err := c.List(context.TODO(), configMaps, client.InNamespace(configMapNamespace)); err != nil {
+		klog.Error(err, "unable to get ConfigMapList")
+		return "", err
+	}
+	if len(configMaps.Items) != 1 {
+		err := fmt.Errorf("there are %d ConfigMaps at the moment, unstable condition", len(configMaps.Items))
```

Comment:
What does "unstable" mean? ATM there are two configmaps in the liqo namespace in our deployment. This piece of code would never work

Reply:
Now I have understood the point, thank you